pages/osx/port.md

```diff
@@ -0,0 +1,27 @@
+# port
+
+> Package manager for macOS.
+
+- Search for package:
+
+`port search {{search_term}}`
+
+- Install package:
+
+`sudo port install {{package_name}}`
+
+- List installed packages:
+
+`port installed`
+
+- Update ports tree:
```

Comment:
Might be better to remove "tree". How about -
```
Update `port` to the latest version
```

Reply:
That's a good point. I think it still needs to refer to the update to the tree in some form though as that's its most common use-case. How about something like:

`Update `port` and fetch latest list of available packages`

Reply:
Maybe I don't understand clearly, what exactly do you mean by "tree" here ?

Reply:
The port tree is what Mac Ports refers to the list of packages as. From their documentation for this command: "The local ports tree is a collection of files that contain information on which packages are available through MacPorts and how they can be installed. You should regularly update your ports tree to get access to updated versions of software and bug fixes. To do that, use selfupdate"

`port selfupdate` also upgrades the port program as well if a new version is available. The output looks something like:

```` --->  Updating MacPorts base sources using rsync
MacPorts base version 2.4.1 installed,
MacPorts base version 2.4.1 downloaded.
--->  Updating the ports tree
--->  MacPorts base is already the latest version

The ports tree has been updated. To upgrade your installed ports, you should run
  port upgrade outdated